repository_service_tuf_worker/repository.py

```diff
@@ -284,8 +284,7 @@ def _update_timestamp(
 
         # TODO review if here is the best place to change the status in DB
```

Comment:
You mentioned this `TODO` in the issue you have linked.
Can we remove it or this is question is still open?

Reply:
I would keep the question still open. We must use it more to see if it is the right/best place.